src/formio/components/map.ts

```diff
@@ -41,6 +41,10 @@ export interface MapComponentSchema
     lat?: number;
     lng?: number;
   };
+  /**
+   * The background identifier to use for the map component
+   */
+  backgroundIdentifier?: string;
```

Comment:
I think `background` is not the optimal name for this, how about `tileLayerIdentifier`? This makes sure we use the same terms as the (leaflet) map itself since it is the tile layer that gets configured with this.

Reply:
Yeah sounds good